models/dao/addresses/arbitrum/dao_addresses_arbitrum_schema.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+models:
+  - name: dao_addresses_arbitrum
+    meta:
+      blockchain: arbitrum
+      sector: dao
+      contributors: Henrystats 
+    config:
+      tags: [ 'arbitrum', 'dao', 'aragon', 'dao creator tools', 'Henrystats' ]
+    description: >
+      wallet addresses of daos created on arbitrum across all dao creator tools
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "blockchain which the DAO is deployed"
+      - &dao_creator_tool
+        name: dao_creator_tool
+        description: "Name of creator tool used to create DAO"
+        tests:
```

Comment:
i don't think we need the seed test here, right?
i believe this seed test is already running in the upstream spell